torch/csrc/api/include/torch/xpu.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <torch/csrc/Export.h>
+
+#include <cstddef>
+#include <cstdint>
+
+namespace torch {
+namespace xpu {
```

Comment:
Use nested namespaces please
```suggestion
namespace torch::xpu {
```

Reply:
Done.